docs/how-to-guides/advanced-user-guides/istio-setup-guide/generate-and-view-traffic.md

```diff
@@ -6,19 +6,21 @@ title: Generate and View Traffic from Istio
   <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/istio-setup-guide/generate-and-view-traffic"/>
 </head>
 
-This section describes how to view the traffic that is being managed by Istio.
+## Prerequisites
 
-## The Kiali Traffic Graph
+To enable traffic to show up in the graph, ensure you have prometheus installed in the cluster. `Rancher-istio` installs Kiali configured by default to work with the `rancher-monitoring` chart. You can use `rancher-monitoring` or install your own monitoring solution.
 
-The Istio overview page provides a link to the Kiali dashboard. From the Kiali dashboard, you are able to view graphs for each namespace. The Kiali graph provides a powerful way to visualize the topology of your Istio service mesh. It shows you which services communicate with each other.
```

Comment:
I agree with making the Prereqs note into its own section, but why get rid of the intro? Users might land on the page from external search.

Reply:
Ok, I see this was moved below prereqs. IMO it should come earlier to give context for the rest of the page

Reply:
Moved to the top of the page.